ui/src/style/pages/admin.scss

```diff
@@ -64,113 +49,81 @@
     }
   }
 }
-
+.admin-tabs--content {
+  .panel {border-top-left-radius: 0;}
+  .panel-heading {height: 60px;}
+  .panel-title {
+    font-size: 17px;
+    font-weight: 400 !important;
+    color: $g12-forge;
+    padding: 0;
+  }
+  .panel-body {min-height: 300px;}
+  .panel-heading + .panel-body {padding-top: 0;}
+}
 
 /*
     Admin Table
     ----------------------------------------------
 */
-.admin-table {
-  .multi-select-dropdown {
-    width: 100%;
-    min-width: 150px;
-  }
-  .admin-table--kill-button {
-    width: 70px;
-  }
-  .admin-table--hidden {
-    visibility: hidden;
-  }
-  .dropdown-toggle {
-    background-color: transparent;
-    font-size: 14px;
-    font-weight: 500;
-    color: $g14-chromium;
-    width: 100%;
-    transition: none !important;
-
-    .caret {opacity: 0;}
-    .multi-select-dropdown__label {left: 0;}
-  }
-  .open .dropdown-toggle .multi-select-dropdown__label {left: 9px;}
-  tbody tr:hover {
-    .admin-table--hidden {
-      visibility: visible;
-    }
-    .dropdown-toggle {
-      color: $g20-white !important;
-      background-color: $c-pool;
-      font-weight: 600;
-
-      .caret {opacity: 1;}
-      .multi-select-dropdown__label {left: 9px;}
-
-      &:hover {
-        transition: background-color 0.25s ease;
-        background-color: $c-laser;
-      }
-    }
-  }
-  .multi-select-dropdown.open .dropdown-toggle {
-    color: $g20-white !important;
-    background-color: $c-laser !important;
-    font-weight: 600;
-    .multi-select-dropdown__label {left: 9x !important;}
-  }
+.admin-table .dropdown-toggle {
+  background-color: transparent;
+  font-weight: 600;
+  color: $g14-chromium;
+  transition: none !important;
+
+  .caret {opacity: 0;}
 }
-.admin-table--edit-row {
-  background-color: $g4-onyx;
+.admin-table--multi-select-empty .dropdown-toggle {
+  color: $g8-storm;
 }
-.admin-table--edit-cell {
-  width: 100%;
-  margin: 0 !important;
-  display: flex !important;
-  justify-content: space-between;
+.admin-table tbody tr:hover .dropdown-toggle {
+  color: $g20-white !important;
+  background-color: $c-pool;
 
-  > input {
-    height: 30px;
-    padding: 0 9px;
-    flex-grow: 1;
-    margin: 0 2px;
-    min-width: 110px;
+  .caret {opacity: 1;}
 
-    &:first-child {margin-left: 0;}
-    &:last-child {margin-right: 0;}
+  &:hover {
+    transition: background-color 0.25s ease;
+    background-color: $c-laser;
   }
 }
+table > tbody > tr > td.admin-table--left-offset,
```

Comment:
What's the benefit of making the nesting chain explicit? Can we simplify to `.admin-table--left-offset`, etc?